### ISSUE TITLE
Use something other than color to indicate that a page has been annotated

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#785]
This issue is part of the larger accessibility epic written up here #753

Problem:
We failed in this accessibility check under the Use of Color section because we don't currently have a a way other than the highlights on a page to indicate that something has been annotated. Yes, we have a badge, but that only appears for users that have the extension. This feature would address notifying users that annotations are present on a page no matter how they arrive to the annotation (embed, bookmarklet, extension). 

Acceptance Criteria: 
- [ ] Create a new way to indicate that annotations have been created on a page other than the highlights appearing on the page. 
- [ ] This feature must be customizable, as our partners currently have the ability to turn highlights on/off. 
- [ ] Show user that a specific section of the page has been annotated, as well as that there are annotations on the page (so denote in some way that a particular passage, image, etc has annotations). 

Note: maybe something to contribute back to W3C standards.

**Comment by dwhly** [comment]
What about the bucket bar?

**Comment by ajpeddakotla** [comment]
Re: Bucket Bar @dwhly 
Currently if you're using a screenreader, it goes through all of the text on the body of the page and then it goes to sidebar. By the time you get to sidebar, you have no idea what the bucket bar #s are referring to, because it will say the # of annotations without giving the precise location of those annotations.

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 3 | 3 | 1 |

**Comment by klemay** [comment]
This was fixed as of https://github.com/hypothesis/client/issues/1817 - closing
